include/uaclient/uasession.h

```diff
@@ -41,15 +41,20 @@ struct SessionSecurityInfo
 
 struct SessionConnectInfo
 {
-    UaString    sApplicationName;
-    UaString    sApplicationUri;
-    UaString    sProductUri;
-    OpcUa_UInt32  internalServiceCallTimeout;
+    UaString     sApplicationName;
```

Comment:
what do you think about making these members const? For that extra immutability vibe.

Reply:
Hi, 
I'm against, because the purpose of open62541-compat is to follow UA-SDK API as close as practically possible. In UA-SDK these are mutable, I think (unless they are immutable, then pinch me).

Reply:
good point, ignore that!